test/mocha/094-xblock-ws-wp.js

```diff
@@ -0,0 +1,230 @@
+/*!
```

Comment:
Let's name this `095-...` because there will be conflict with this already existing PR:

https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/pull/257

This test may also be experiencing a hang / issues because it lacks the fix here:

https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/pull/257/files#diff-9a0d33be264e030d26b5348338cf6d05e56023e29e1a81ef7ec7a9584b2bde45

But this may be avoidable once a witness pool doc is actually written.

Reply:
Done in https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/pull/260/commits/26193890cb9ff732faad25d08151bdc25987ad74